addon/components/render-mobiledoc.js

```diff
@@ -99,8 +99,7 @@ export default Ember.Component.extend({
   }),
 
   _renderMobiledoc() {
-    let domHelper = getDOM(this);
-    let dom = domHelper.document;
+    let dom = getDOM(this);
```

Comment:
`getDOM` seems to be a misnomer with your refactor. The `document` is being returned. Perhaps this API should be renamed `getDocument`?